sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala

```diff
@@ -25,8 +25,13 @@ import org.apache.spark.util.{Benchmark, Utils}
 
 /**
  * The benchmarks aims to measure performance of JSON parsing when encoding is set and isn't.
- * To run this:
- *  spark-submit --class <this class> --jars <spark sql test jar>
+ * To run:
+ *  mvn clean package -pl sql/core -DskipTests
+ *  ./dev/make-distribution.sh --name local-dist
+ *  cd dist/
+ *  ./bin/spark-submit --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks \
+ *      ../sql/core/target/spark-sql_2.11-2.4.0-SNAPSHOT-tests.jar > /tmp/output.txt
```

Comment:
I think it's better to have more specific commands but let's avoid to have "2.4.0-SNAPSHOT" in this. Otherwise, we should change this when we make another release ..

Reply:
I agree, but running this benchmark is quite specific and should only happend when something is changed with the json functionality. Not sure if you want to have a specific build step for it.

Reply:
But wouldn't this command be invalid if we have another release. I mean, I believe we can build by following https://spark.apache.org/docs/latest/building-spark.html and run the benchmark.

Let's just say it like `bin/spark-submit --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks --jars ./sql/core/target/spark-sql_*.jar` or like `spark-submit --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks --jars <spark sql test jar>`

Should better to make the steps independent of other possible ways.

Reply:
Ok, I'm fine with that. I've updated the comments.

Reply:
Eh, seems missed to update?

Reply:
Oops, forgot to add:
```
MacBook-Pro-van-Fokko:spark fokkodriesprong$ git diff
diff --git a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala
index 3b273f81d7..0598406d33 100644
--- a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala
+++ b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala
@@ -29,8 +29,9 @@ import org.apache.spark.util.{Benchmark, Utils}
  *  mvn clean package -pl sql/core -DskipTests
  *  ./dev/make-distribution.sh --name local-dist
  *  cd dist/
- *  ./bin/spark-submit --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks \
- *      ../sql/core/target/spark-sql_2.11-2.4.0-SNAPSHOT-tests.jar > /tmp/output.txt
+ *  ./bin/spark-submit bin/spark-submit \
+ *      --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks \
+ *      --jars ./sql/core/target/spark-sql_*-tests.jar
```

Reply:
Let's take out other comments like `make-distribution.sh` and `cd dist/` too since they can be varied by how to build.